cypress/e2e/communityTemplates.test.ts

```diff
@@ -72,6 +80,37 @@ describe('Community Templates', () => {
       )
     })
 
+    it('can install using the keyboard "Enter" key and without clicking "Lookup Template"', () => {
+      //The lookup template accepts github raw link
+      cy.getByTestID('lookup-template-input').type(
+        'https://raw.githubusercontent.com/influxdata/influxdb/master/pkger/testdata/dashboard_band.yml'
```

Comment:
gotta promote / test your own work, right? 😏

Reply:
It's an easter egg. And you found it!